packages/@aws-cdk/cdk/lib/reference.ts

```diff
@@ -1,116 +1,26 @@
-import { ResolveContext, Token } from "./token";
+import { Token } from "./token";
 
 /**
- * A Token that represents a CloudFormation reference to another resource
+ * A Token that represents a reference between two constructs
  *
- * If these references are used in a different stack from where they are
- * defined, appropriate CloudFormation `Export`s and `Fn::ImportValue`s will be
- * synthesized automatically instead of the regular CloudFormation references.
- *
- * Additionally, the dependency between the stacks will be recorded, and the toolkit
- * will make sure to deploy producing stack before the consuming stack.
- *
- * This magic happens in the prepare() phase, where consuming stacks will call
- * `consumeFromStack` on these Tokens and if they happen to be exported by a different
- * Stack, we'll register the dependency.
+ * References are recorded.
  */
 export class Reference extends Token {
   /**
    * Check whether this is actually a Reference
    */
   public static isReferenceToken(x: Token): x is Reference {
```

Comment:
`isReference`?

Reply:
I'm all for it, but the previous implementor (you?) picked this name.